mne/io/edf/edf.py

```diff
@@ -392,6 +391,7 @@ def _get_edf_info(fname, stim_channel, annot, annotmap, eog, misc, exclude,
                  ' Inferring from the file size.')
             edf_info['n_records'] = n_records = read_records
 
+    n_samps = n_samps[include]
```

Comment:
This reduction was done at line 378 before. Now it is done after the computation of ``read_records = total_samps // np.sum(n_samps)`` (line 387).